retirement_api/tests/test_commands.py

```diff
@@ -0,0 +1,17 @@
+import mock
+import unittest
+
+from django.core.management import call_command
+
+from retirement_api.management.commands import check_ssa_values
+
+
+class CommandTests(unittest.TestCase):
+    @mock.patch('retirement_api.management.commands.check_ssa_values.ssa_check.run_tests')
+    def test_check_ssa_values(self, mock_run_tests):
+        mock_run_tests.return_value = 'OK'
+        test_run = call_command('check_ssa_values')
+        self.assertTrue(mock_run_tests.call_count == 1)
+        test_run2 = call_command('check_ssa_values',
+                                 '--recalibrate')
+        self.assertTrue(mock_run_tests.call_count == 2)
```

Comment:
You can also check the standard output to see if it prints out what you wanted.

I used it somewhere in a project but I can't find it at this point, so I find some sample out there:
http://stackoverflow.com/questions/4219717/how-to-assert-output-with-nosetest-unittest-in-python, I believe I used the one with 2 upvotes at the bottom.
```
from io import StringIO
from unittest.mock import patch

with patch('sys.stdout', new=StringIO()) as fakeOutput:
    print('hello world')
    self.assertEqual(fakeOutput.getvalue().strip(), 'hello world')
```